CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # HEAD
 
+# 1.3.4
+
+Improving
+
+* Added `Chain#last` method (@andrykonchin)
+* Added `date` field type (@andrykonchin)
+* Added `application_timezone` config option (@andrykonchin)
+* Allow consistent reading for Scan request (@andrykonchin)
+* Use Query instead of Scan if there are no conditions for sort (range) key in where clause (@andrykonchin)
+* Support condition operators for non-key fields for Query request (@andrykonchin)
+* Support condition operators for Scan request (@andrykonchin)
+* Support additional operators `in`, `contains`, `not_contains` (@andrykonchin)
+* Support type casting in `where` clause (@andrykonchin)
+* Rename `Chain#eval_limit` to `#record_limit`  (@richardhsu)
+* Add `Chain#scan_limit`  (@richardhsu)
+* Support batch loading for Query requests (@richardhsu)
+* Support querying Global/Local Secondary Indices  (@richardhsu)
```

Comment:
* Support querying Global/Local Secondary Indices in `where` clause (@richardhsu)